server/controllers/AuthController.js

```diff
@@ -85,7 +85,7 @@ class AuthController {
       } else {
         const user = result.rows[0];
         const userInfo = {
-          id: user.id,
+          id: user.acct_id,
```

Comment:
What is acct_id? Would you rather use a more descriptive name? Also, why are you using snake case? It is not a convention in Javascript.